components/03-modules/modal/modal.hbs

```diff
@@ -1,14 +1,28 @@
 {{#if trigger }}
-    {{ render '@modal-trigger' }}
+    {{ render '@modal-trigger' modalTrigger }}
 {{/if}}
-<div class="modal {{ modalClass }}" data-modal="{{ modalId }}">
-    <dialog class="modal__content {{ dialogClass }}" {{{ dialogAttribtues }}}>
-        {{ render (component content) contentContext }}
+<div role="dialog"
+     aria-hidden="true"
+     id="{{ modal.id }}"
+     data-modal="{{ modal.id }}"
+     class="modal {{ modal.class }}"
+     tabindex="-1"
+     {{{ modal.attributes }}}
+ >
+    <div role="document"
+         class="modal__content {{ modalContent.class }}"
+         {{{ modalContent.attribtues }}}
+         tabindex="0"
```

Comment:
please remove tabindex from this element, it's not necessary, in the variant template too

Reply:
removed but as later I mentioned that maybe because of UX is better to leave it we revert it back.